modules/integration/src/test/scala/scala/cli/integration/NativePackagerTests.scala

```diff
@@ -430,7 +430,8 @@ class NativePackagerTests extends ScalaCliSuite {
   }
 
   if (Properties.isLinux)
-    test("building docker image with scala native app") {
+    // FIXME this got flaky on the CI again
+    test("building docker image with scala native app".flaky) {
```

Comment:
Do we have an open issue for this?

Reply:
yes, I mentioned it in the PR description.
I reopened this:
- https://github.com/VirtusLab/scala-cli/issues/3271